### PR TITLE
Positioncard - future and past positions

### PR DESCRIPTION
**Pull request by ingridklepsvik**

### CODE REVIEW

**Comment by ingridklepsvik** [comment]
https://dev.azure.com/statoil-proview/Fusion/_workitems/edit/8167/

File: src/components/org/PositionCard/components/PositionInstance.tsx [hunk 1]
```diff
@@ -21,6 +23,8 @@ type PositionInstanceProps = {
 const PositionInstanceComponent: React.FC<PositionInstanceProps> = ({
     position,
     instance,
+    isFuture,
```
Comment (eskil0312) [inline src/components/org/PositionCard/components/PositionInstance.tsx]: Do we need these props in the Instance component?

File: src/components/org/PositionCard/components/PositionInstance.tsx [hunk 2]
```diff
@@ -8,6 +8,8 @@ import PositionTimeline from './PositionTimeline';
 type PositionInstanceProps = {
     position: Position;
     instance?: PositionInstance;
+    isFuture?: boolean;
```
Comment (eskil0312) [inline src/components/org/PositionCard/components/PositionInstance.tsx]: Do we need these props in the Instance component?

File: src/components/org/PositionCard/components/PositionInstance.tsx [hunk 3]
```diff
@@ -43,7 +47,7 @@ const PositionInstanceComponent: React.FC<PositionInstanceProps> = ({
     const directChildrenTooltipRef = useTooltipRef(`${childCount} positions`, 'above');
     const externalIdTooltipRef = useTooltipRef('External ID: ' + position.externalId, 'below');
 
-    const positionInstanceClasses = classNames(styles.positionInstance, {
+    const positionInstanceClasses = classNames(styles.positionInstance, styles.content, {
```
Comment (eskil0312) [inline src/components/org/PositionCard/components/PositionInstance.tsx]: What is styles.content and is it needed?

File: src/components/org/PositionCard/components/PositionInstance.tsx (hunk 1 above)
Comment (ingridklepsvik) [reply src/components/org/PositionCard/components/PositionInstance.tsx]: Removed

File: src/components/org/PositionCard/components/PositionInstance.tsx (hunk 3 above)
Comment (ingridklepsvik) [reply src/components/org/PositionCard/components/PositionInstance.tsx]: Removed

File: src/components/org/PositionCard/components/PositionInstance.tsx (hunk 2 above)
Comment (ingridklepsvik) [reply src/components/org/PositionCard/components/PositionInstance.tsx]: Removed

**Pull request merged**